hashicorp/dev-portal
# feat: adds new edition options

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksadds-new-tfc-edition-options-hashicorp.vercel.app/terraform/tutorials/cloud/cloud-agents) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1203675527903646) 🎟️

## 🗒️ What

This PR adds two new edition types to align with Terraforms upcoming pricing changes. The legacy types are there so we can ease the migration as authors go to update all source content with the new editions. 

[This follow up task](https://app.asana.com/0/1203652232454030/1203704507579626) involves removing the old options once the content is updated by March. 

See related PRs:
- https://github.com/hashicorp/learn-api-content-sync/pull/67
- https://github.com/hashicorp/learn-api/pull/146

## To test

There is no way to test on the preview since the content doesn't use these editions. I seeded a local api / database with the updates in those backend PRs, and tested the frontmatter with the new edition types. Here is a picture of the new options in use (see badges list in the tutorial meta): 
 
<img width="1313" alt="Screen Shot 2023-01-11 at 4 08 48 PM" src="https://user-images.githubusercontent.com/36613477/211944811-cfde6381-ffe1-4ae9-8638-328782203641.png">

<img width="1218" alt="Screen Shot 2023-01-11 at 4 07 45 PM" src="https://user-images.githubusercontent.com/36613477/211944803-c537f218-e7c3-490e-b6bd-b5ccb8a76643.png">



However, all the other edition types should still work. And there should be no changes to[ the tutorial view.](https://dev-portal-git-ksadds-new-tfc-edition-options-hashicorp.vercel.app/terraform/tutorials/cloud/cloud-agents)
